src/OpenTelemetry/Metrics/MeterProviderBuilderExtensions.cs

```diff
@@ -0,0 +1,55 @@
+// <copyright file="MeterProviderBuilderExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Metrics
+{
+    /// <summary>
+    /// Contains extension methods for the <see cref="MeterProviderBuilder"/> class.
+    /// </summary>
+    public static class MeterProviderBuilderExtensions
+    {
+        /// <summary>
+        /// Sets observation period.
+        /// </summary>
+        /// <param name="meterProviderBuilder"><see cref="MeterProviderBuilder"/>.</param>
+        /// <param name="periodMilliseconds">Perion in milliseconds.</param>
+        /// <returns><see cref="MeterProvider"/>.</returns>
+        public static MeterProviderBuilder SetObservationPeriod(this MeterProviderBuilder meterProviderBuilder, int periodMilliseconds)
```

Comment:
Not a blocker for this PR.
I guess we will struggle with the ObservationPeriod since there are scenarios where different instruments being observed and reported at different frequency (e.g. during surgery, temperature is observed and reported every minute, blood pressure is observed and reported every 0.5 second, heart beat is observed every 0.1 second but reported every 15 seconds).

Reply:
agree. Will keep this now to continue experimenting with this.